MekHQ/src/mekhq/campaign/personnel/education/EducationController.java

```diff
@@ -341,33 +348,16 @@ public static void processNewDay(Campaign campaign) {
      * @param resources The resource bundle containing localized strings.
      * @return True if the person's journey to campus was processed, false otherwise.
      */
-    private static boolean journeyToAcademy(Campaign campaign, Person person, ResourceBundle resources) {
+    private static void journeyToAcademy(Campaign campaign, Person person, ResourceBundle resources) {
         int daysOfTravelTo = person.getEduDaysOfTravelToAcademy();
 
-        if (daysOfTravelTo > 0) {
-            person.setEduDaysOfTravelToAcademy(daysOfTravelTo - 1);
+        person.setEduDaysOfTravelToAcademy(daysOfTravelTo - 1);
 
-            // has Person just arrived?
-            if ((daysOfTravelTo - 1) == 0) {
-                campaign.addReport(person.getHyperlinkedName() + ' ' + resources.getString("arrived.text"));
-            }
-
-            return true;
+        // has Person just arrived?
+        if ((daysOfTravelTo - 1) == 0) {
```

Comment:
Is it possible for `daysOfTravelTo - 1` to be less than 0?  If so, might want to adjust the comparison.

Reply:
Shouldn't be possible, but I can add a catcher, just in case. One sec :)